zospy/analyses/base.py

```diff
@@ -324,9 +324,8 @@ def Results(self) -> _ZOSAPI.Analysis.Data.IAR_:  # noqa: N802
         """Analysis results."""
         return self._analysis.GetResults()
 
-    @property
-    def field(self) -> int | str:
-        """Gets the wavelength value of the analysis.
+    def get_field(self) -> int | str:
```

Comment:
```suggestion
    def get_field(self) -> int | Literal["All"]:
```

Furthermore, please put the `get_field` and `set_field` methods between the properties and `(get|set)_wavelength`.